kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/matchers/collections/sortedDesc.kt

```diff
@@ -0,0 +1,58 @@
+package io.kotest.matchers.collections
```

Comment:
this all but destroys git history, which is very valuable, so there should be a very strong reason for doing so. what is it?

Reply:
The reason I moved this into its own file is that there are generally files whose name corresponds to the assertion it contains. With the notable exception of the old `CollectionMatchers.kt` and `matchers.kt` files that contain a mishmash of matchers. 

What I observed so far is that each named file contains the following:
1) A set of assertions - extensions on a container
2) Negations of those assertions
3) Matchers implementing the assertion algorithm, utilized by the extensions

It seems to me that the file as a unit of organization contains a _single set_ of these assertions/matchers i.e. a single algorithm.
`Sorted` and `SortedDesc` are currently two separate algorithms with two sets of matchers, not one matcher.

Two sets of matchers to me indicates (according to the observed pattern) that these assertions should live in standalone files, such that each file contains its own respective algorithm and assertion set.

What do you think? Is it a reasonable observation? There have been similar breakouts in the past where an assertion was given its own file.

Reply:
There are files `containExactly.kt` and `containExactlyInAnyOrder.kt` - two similar-yet-different assertions. `sorted.kt` and `sortedDesc.kt` seems to be of similar kind to me.

Reply:
Please note I'm also a big fan of good history - if I didn't know that all PRs get squashed, this very change would be a set of smaller changes. First colocation of the matchers, then reordering, then breakout. One small step at a time. But as it gets squashed, there wouldn't be much point to it.